src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -1,35 +1,29 @@
 import React, { useCallback, useEffect, useMemo, useState } from "react";
+
+// Typings
```

Comment:
Let’s get rid of these comments. This is a pattern that went against our standards that we want to remove across the codebase, definitely not add more of.